packages/ui/src/Button.tsx

```diff
@@ -178,7 +178,16 @@ export const Button = forwardRef<HTMLElement, ButtonProps>(
                   color={iconLeftColor}
```

Comment:
I'm wondering why we need both the `disabledTooltip` and the `disabledTooltipVisible` props. When we don't provide `disabledTooltip` prop then the `Tooltip` just doesn't render anything. @jankoritak @jvorcak Do you remember why we have to provide `disabledTooltipVisible` explicitly like this? Is this maybe an optimization?
If we just had the `disabledTooltip` prop it would simplify the flow.